modules/apps/object/object-test/src/testFunctional/tests/ObjectRelationships.testcase

```diff
@@ -971,6 +971,83 @@ definition {
 		}
 	}
 
+	@description = "LPS-151676 - Verify that it's possible to edit the relationships of the Custom Object entry (One-to-Many)"
+	@priority = "4"
+	test CanEditOneToManyRelationshipOfCustomObjectEntries {
+		task ("Given some Custom Object entries that contain a Custom Field and Relationship (One-to-many)") {
+			ObjectAdmin.addObjectViaAPI(
+				labelName = "Custom Object 156677",
+				objectName = "CustomObject156677",
+				pluralLabelName = "Custom Objects 156677");
+
+			ObjectAdmin.addObjectFieldViaAPI(
+				fieldBusinessType = "Text",
+				fieldLabelName = "Custom Field",
+				fieldName = "customObjectField",
+				fieldType = "String",
+				isRequired = "false",
+				objectName = "CustomObject156677");
+
+			Navigator.openURL();
+
+			ObjectAdmin.addObjectRelationshipViaAPI(
+				objectName_1 = "CustomObject156677",
+				objectName_2 = "CustomObject156677",
+				relationshipLabel = "Relationship",
+				relationshipName = "relationship",
+				relationshipType = "oneToMany");
+
+			task ("Define custom field as title field on the object") {
+				ObjectAdmin.openObjectAdmin();
+
+				ObjectPortlet.selectCustomObject(label = "Custom Object");
+
+				CreateObject.selectTitleField(fieldLabel = "Custom Field");
+
+				CreateObject.saveObject();
+
+				ObjectAdmin.publishObjectViaAPI(objectName = "CustomObject156677");
+			}
+
+			task ("Add Entrys via API") {
+				for (var fieldEntry : list "A,B,C") {
+					ObjectAdmin.addObjectSingleFieldEntryViaAPI(
+						fieldName = "customObjectField",
+						objectName = "CustomObject156677",
+						value = "Entry ${fieldEntry}");
+				}
+			}
+		}
+
+		task ("And given a relationship is selected for entries") {
+			ObjectAdmin.goToCustomObject(objectName = "CustomObject156677");
+
+			ObjectPortlet.viewEntryDetails(entry = "Entry B");
+
+			ObjectPortlet.chooseEntryOnRelationshipField(entryOption = "Entry A");
+
+			CreateObject.assertEntrySelectedOnRelationshipField(entry = "Entry A");
+
+			PortletEntry.save();
+		}
+
+		task ("When a Custom Object entry is edited") {
+			ObjectAdmin.goToCustomObject(objectName = "CustomObject156677");
+
+			ObjectPortlet.viewEntryDetails(entry = "Entry B");
+
+			ObjectPortlet.selectRelationshipFieldEntry(
+				entry = "Entry A",
+				newEntry = "Entry C");
+		}
+
+		task ("Then the Custom Object entry is updated") {
```

Comment:
Can you add a step to navigate back to the custom object entries page and assert. Want to make sure changes persist